Detector/DetFCCeeCommon/compact/Beampipe_with_notch_noShield.xml

```diff
@@ -0,0 +1,199 @@
+<lccdd>
```

Comment:
This file should go as well to FCCee_o1_v05

Reply:
I think the beampipe should common for all of the experiments, so it should not go in the DetFCCeeCLD folder but stay in DetFCCeeCommon, right?